gatsby-theme-mds/src/components/Colors/Colors.css

```diff
@@ -0,0 +1,18 @@
+.container {
+  border-radius: var(--spacing-m);
+}
+
+.copy-clipboard {
+  color: var(--white);
+  background: var(--inverse-light);
+  padding: var(--spacing-m);
+  border-radius: var(--spacing-m);
+}
+
+.overflow-visible {
+  overflow: visible !important;
```

Comment:
Try to increase the specificity of the class rather than using important.
[Click here to read about it](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)